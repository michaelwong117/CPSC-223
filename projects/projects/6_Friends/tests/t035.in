1,500
3,698
745,15
