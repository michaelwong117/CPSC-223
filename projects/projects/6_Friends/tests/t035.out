1, 500 have minimum path length 11
3, 698 have minimum path length 11
745, 15 have minimum path length 12
